### DISCUSSION TITLE
Corrupt links in project storage view

### DISCUSSION THREAD

**Discussion by trm217** [vercel/community#6440] [Bug Report]
### Page to Investigate

`https://vercel.com/<user>/<project>/stores
`
### Steps to Reproduce

![CleanShot 2024-04-08 at 10 08 23@2x](https://github.com/vercel/community/assets/30313631/5cb451bd-09c8-4411-8506-e8e1554c5a6e)
The href-attribute for the links to the different storages is invalid and looks like
`href="//<project>/stores/<store-type>/<slug>/items"`
For me this happens on both edge-config as well as kv, I assume other storage types have the same issue.

Edit: Also occurs on `https://vercel.com/<user>/~/stores`

**Comment by trm217** [comment]
@amyegan @swarnava @sebastianroming severe UI bug, that prevents accessing the storage services unless you have the direct link. (I was attempting to pitch using vercel storage, but I was unable to due to this bug.

**Reply by sebastianroming** [reply]
@trm217 Thank you for reporting, we're already on it.

**Reply by sebastianroming** [reply]
@trm217 Issue should've been fixed now. Can you give it a try and confirm?

**Reply by PassiDel** [reply]
It works for me again!

**Reply by trm217** [reply]
Works, thank you!
